modules/channel-vonage/src/renderers/audio.ts

```diff
@@ -0,0 +1,35 @@
+import { ChannelRenderer } from 'common/channel'
+import { formatUrl } from 'common/url'
+import { CHANNEL_NAME } from '../backend/client'
+import { VonageContext } from '../backend/typings'
+
+export class VonageAudioRenderer implements ChannelRenderer<VonageContext> {
+  get channel(): string {
+    return CHANNEL_NAME
+  }
+
+  get priority(): number {
+    return 0
+  }
+
+  get id(): string {
+    return VonageAudioRenderer.name
+  }
+
+  handles(context: VonageContext): boolean {
+    return !!context.payload.audio
+  }
+
+  async render(context: VonageContext) {
+    // TODO : const payload = context.payload as sdk.AudioContent
+    const payload = context.payload
+
+    context.messages.push({
+      type: 'audio',
+      text: undefined,
+      audio: {
+        url: formatUrl(context.botUrl, payload.audio)
+      }
+    })
```

Comment:
As explained in the comments of [this PR](https://github.com/botpress/botpress/pull/4927), I think it would be better if instead of having an `'audio'` property that contains a `url` and maybe a `name`, to have both properties at the root of the payload. What do you think?

Reply:
So we said this would require a migration and take too long so we'll look back into it but not right now